tests/unit/NetworkTest.ts

```diff
@@ -35,15 +38,27 @@ const originalXHR = HttpUtils.xhr;
 beforeEach(() => {
     global.fetch = TestHelper.getGlobalFetchMock();
     HttpUtils.xhr = originalXHR;
+
+    // Reset any pending requests
     MainQueue.clear();
     HttpUtils.cancelPendingRequests();
     NetworkStore.checkRequiredData();
-
-    // Wait for any Log command to finish and Onyx to fully clear
-    return waitForBatchedUpdates()
-        .then(() => PersistedRequests.clear())
-        .then(() => Onyx.clear())
-        .then(waitForBatchedUpdates);
+    NetworkStore.setIsAuthenticating(false);
+    resetReauthentication();
```

Comment:
I added `resetReauthentication` because `isAuthenticating` was not being resolved between the tests and polluting others tests. Same for `SequentialQueue.resetQueue`, `isReadyPromise` was not resolving between the tests.